transform/app/api/routes/solr_utils/solr.py

```diff
@@ -0,0 +1,65 @@
+from fastapi import APIRouter, Query
+from datetime import datetime
+import requests
+
+from app.transform.utils.loader import load_env_vars
+from app.transform.schemas.properties.env import (
+    SOLR_ADDRESS,
+    SOLR_PORT,
+    SOLR_COLLECTION_NAMES,
+)
+
+env_vars = load_env_vars()
+router = APIRouter()
+
+
+@router.post("/create_collections")
+async def create_solr_collections(
+    all_collection_config: str = Query(
+        ...,
+        description="Config name for 'all_collection'",
+        example="all_collection_oag56_v203",
+    ),
+    provider_config: str = Query(
+        ..., description="Config name for 'provider'", example="provider_v101"
+    ),
+    collection_prefix: str = Query(
+        "",
+        description="Prefix for collection names. "
+        "It is recommended to use convention 'oag<dump_version>'",
+        example="oag56",
```

Comment:
Lets delete an example also here. Its valid only for one month, later it would be only painful. Empty string as default is okay

Reply:
You're right, I assumed it worked differently. Corrected.